Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -34,6 +34,7 @@ In order to use the CrowdStrike Falcon integration, your API client must be prov
 - Alerts - Read and Write
 - Hosts - Read
 - IOC Manager - Read and Write
+- On-Demand Scans (ODS) - Read and Write
```

Comment:
need to generate readme here. We should add a command to the command examples file (examples.txt) to do it. lets discuss

Reply:
will do